homework01/rudolf/vector.h

```diff
@@ -0,0 +1,31 @@
+#ifndef __VECTOR_H_INCLUDED__   // if x.h hasn't been included yet...
+#define __VECTOR_H_INCLUDED__   //   #define this so the compiler knows it has been included
+
+#include <iostream>
```

Comment:
Please remove this include, this Vector class does not depend on anything from it.